RedditSharp/Things/Subreddit.cs

```diff
@@ -516,6 +614,20 @@ public void AddModerator(string user)
             var response = request.GetResponse();
             var result = WebAgent.GetResponseString(response.GetResponseStream());
         }
+        public void AddModerator(RedditUser user)
```

Comment:
I'm willing to add this in, but it's probably unnecessary because you can get the username string trivially from the RedditUser object so there isn't much point in the overload